website/src/pages/Home/index.tsx

```diff
@@ -192,7 +157,7 @@ const Home: React.FC = () => {
             white_logo: "../Images/logos/mongodb_white.svg"
         },
         Prometheus : {
-            logo: "../Images/logos/prometheus.svg",
+            logo: "../Images/logos/Prometheus.svg",
```

Comment:
usually, the name of assets should be LowerCase. (consistency)

Reply:
also, can we rename `Images` to `images`? Since there reside assets in this directory, not any react components.